### PR TITLE
fix(#4970) : font-family

### PR DESCRIPTION
**Pull request by Blakelist7**
Some of the font-family containing whitespaces were not quoted

Quoted font-family having whitespaces.

#4970

### CODE REVIEW

**Comment by wbamberg** [comment]
To fix this bug I think the minimum is that anywhere a family name containing spaces is given as example code, it should be quoted.

This means [line 27](https://github.com/mdn/content/blob/ca3b41e998c548e46c17f5918265201ea8343102/files/en-us/web/css/font-family/index.html#L29) and [line 57](https://github.com/mdn/content/blob/ca3b41e998c548e46c17f5918265201ea8343102/files/en-us/web/css/font-family/index.html#L57) (though actually I don't even know why this additional example is included at all, it just duplicates part of the bigger block above).

This does not include places where names are just mentioned in the text, like in lines [69-71](https://github.com/mdn/content/blob/ca3b41e998c548e46c17f5918265201ea8343102/files/en-us/web/css/font-family/index.html#L69-L71), because that's just referencing the font family, it isn't providing example code.

The bit at [lines 108-111](https://github.com/mdn/content/blob/ca3b41e998c548e46c17f5918265201ea8343102/files/en-us/web/css/font-family/index.html#L108-L111) is a special case, because the declaration at line 110 *is* valid, it's just not recommended. So it would be better to clarify the situation in this section, by saying something like:

> Font family names must either be given quoted as strings, or unquoted as a sequence of one or more identifiers. This means that punctuation characters and digits at the start of each token must be escaped in unquoted font family names.
> 
> It is good practice to quote font family names that contain white space, digits, or punctuation characters other than hyphens. 

...then in the examples, we could make the "good" example (coloured green) only include the "font-family: "Goudy Bookletter 1911", sans-serif;" example, and add a third example after the "bad" example, that should be neither green nor red, but precede it like:

> The following example is technically valid but is not recommended:
>
>   ```css
>  font-family: Gill Sans Extrabold, sans-serif;
>   ```

Does that make sense?

**Comment by Blakelist7** [comment]
Yeah I think that do make some sense.

**Comment by Blakelist7** [comment]
Required changes done :)

File: files/en-us/web/css/font-family/index.html [hunk 1]
```diff
@@ -54,13 +54,13 @@ <h2 id="Syntax">Syntax</h2>
...
 
-<pre class="brush: css;">font-family: Gill Sans Extrabold, sans-serif;</pre>
+<pre class="brush: css;">font-family: "Gill Sans Extrabold", sans-serif;</pre>
 
 <h3 id="Values">Values</h3>
 
 <dl>
  <dt><a id="family-name"><code>&lt;family-name&gt;</code></a></dt>
- <dd>The name of a font family. For example, "Times" and "Helvetica" are font families. Font family names containing whitespace should be quoted.</dd>
+ <dd>The name of a font family. For example, "Times" and "Helvetica" are font families. Font family names containing whitespace should be quoted. For example, 'Comic Sans MS'.</dd>
```
Comment (wbamberg) [inline files/en-us/web/css/font-family/index.html:63]:
```suggestion
 <dd>The name of a font family. For example, "Times" and "Helvetica" are font families. Font family names containing whitespace should be quoted. For example: "Comic Sans MS".</dd>
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
I had one copy-edit comment, and it looks like we missed https://github.com/mdn/content/blob/f9d710480812705f184665a3bd42de55e9ab098f/files/en-us/web/css/font-family/index.html#L136 and https://github.com/mdn/content/blob/f9d710480812705f184665a3bd42de55e9ab098f/files/en-us/web/css/font-family/index.html#L144 . Sorry!

**Comment by Blakelist7** [comment]
> I had one copy-edit comment, and it looks like we missed
> 
> https://github.com/mdn/content/blob/f9d710480812705f184665a3bd42de55e9ab098f/files/en-us/web/css/font-family/index.html#L136
> and
> 
> https://github.com/mdn/content/blob/f9d710480812705f184665a3bd42de55e9ab098f/files/en-us/web/css/font-family/index.html#L144
> . Sorry!

No worry. Next time I'll be more careful 😃 
Done with changes :)

**Comment by Blakelist7** [comment]
@wbamberg done with suggestions :D

File: files/en-us/web/css/font-family/index.html [hunk 2]
```diff
@@ -125,7 +125,7 @@ <h3 id="Valid_family_names">Valid family names</h3>
 <p>The following example is technically <strong>valid</strong> but is not recommended:</p>
 
 <pre class="brush: css;">
-font-family: Gill Sans Extrabold, sans-serif;
+font-family: "Gill Sans Extrabold", sans-serif;
```
Comment (wbamberg) [inline files/en-us/web/css/font-family/index.html:128]: This should *not* be quoted, because it's an example of the not-good practice of not quoting.

Comment (Blakelist7) [reply files/en-us/web/css/font-family/index.html]: My bad, did'nt saw that

**Comment by Blakelist7** [comment]
@wbamberg now ig it's all clear 👀

**Review by wbamberg — APPROVED** [review]
Thanks @Blakelist7 , looks good to me!

**Pull request merged**